clients/src/main/java/org/apache/kafka/common/requests/ListOffsetResponse.java

```diff
@@ -191,4 +191,15 @@ protected Struct toStruct(short version) {
 
         return struct;
     }
+
+    @Override
+    public String toString() {
+        StringBuilder bld = new StringBuilder();
+        bld.append("(type=ListOffsetResponse")
+            .append(", throttleTimeMs=").append(throttleTimeMs)
+            .append(", responseData=").append(responseData);
+
+        bld.append(")");
```

Comment:
*Really* minor nit, but can we chain this `append` with all the others like we do in other implementations of request/response `toString()` methods?